test.sh

```diff
@@ -16,7 +16,34 @@
 
 # Simple shell script to lint files and run the tests. Could be helpful for
 # users, but largely for automation.
+#
+# NOTE! If this file changes/moves, please change
+# https://github.com/kubernetes/test-infra/blob/master/jobs/config.json
+
+# Ensure we're running from this directory to ensure PYTHONPATH is set
+# correctly.
+cd "$(dirname "$0")"
+
+lint_fail=0
+ls *.py | xargs pylint || {
```

Comment:
this is different than autopep8?

Reply:
This is linting. I think autopep8 is just a formatter?